components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java

```diff
@@ -337,7 +349,13 @@ public List selectComponentsLastMetric(String carbonId, String appName, Map<Stri
         return componentsRecentMetrics;
     }
 
-
+    public static String humanReadableByteCount(double bytes, boolean si) {
```

Comment:
If this is not library-derived code, can you please add javadoc comments for readability?